test/orange/univ/SimpleUnitTracker.test.cc

```diff
@@ -299,9 +301,15 @@ auto SimpleUnitTrackerTest::reduce_heuristic_init(StateHostRef const& host,
     std::vector<size_type> counts(this->num_volumes());
     size_type error_count{};
 
-    for (auto vol : host.vol[AllItems<VolumeId>{}])
+    for (auto i : range(host.size()))
     {
-        if (vol < counts.size())
+        auto thread_id = ThreadId{i};
+        // TODO Update for multiple universes
+        // LevelStateAccessor lsa(&host, thread_id)
+        LevelStateAccessor lsa(&host, thread_id, LevelId{0});
+        auto vol = lsa.vol();
+
+        if (vol.unchecked_get() < counts.size())
```

Comment:
```suggestion
        if (vol < counts.size())
```
This will be `false` if `vol` is unset *or* if it's out of range.